connection_test.go

```diff
@@ -398,70 +398,49 @@ func TestPrivateLink(t *testing.T) {
 }
 
 func TestGetQueryStatus(t *testing.T) {
-	config, err := ParseDSN(dsn)
-	if err != nil {
-		t.Error(err)
-	}
-	ctx := context.Background()
-	sc, err := buildSnowflakeConn(ctx, *config)
-	if err != nil {
-		t.Error(err)
-	}
-	if err = authenticateWithConfig(sc); err != nil {
-		t.Error(err)
-	}
+	runSnowflakeConnTest(t, func(sct *SCTest) {
+		ctx := sct.sc.ctx
 
-	if _, err = sc.Exec(`create or replace table ut_conn(c1 number, c2 string)
+		sct.mustExec(`create or replace table ut_conn(c1 number, c2 string)
 		as (select seq4() as seq, concat('str',to_varchar(seq)) as str1 from
-		table(generator(rowcount => 100)))`, nil); err != nil {
-		t.Error(err)
-	}
+		table(generator(rowcount => 100)))`, nil)
 
-	rows, err := sc.QueryContext(ctx, "select min(c1) as ms, sum(c1) from ut_conn group by (c1 % 10) order by ms", nil)
-	if err != nil {
-		t.Error(err)
-	}
-	qid := rows.(SnowflakeResult).GetQueryID()
+		rows, err := sct.sc.QueryContext(ctx, "select min(c1) as ms, sum(c1) from ut_conn group by (c1 % 10) order by ms", nil)
```

Comment:
Maybe we can introduce `mustQueryContext` here?

Reply:
Not in this case. We need error here.